test/integration/aaa_download_only_test.go

```diff
@@ -74,7 +74,7 @@ func TestDownloadOnly(t *testing.T) {
 
 				for _, img := range imgs {
 					img = strings.Replace(img, ":", "_", 1) // for example kube-scheduler:v1.15.2 --> kube-scheduler_v1.15.2
-					fp := filepath.Join(localpath.MiniPath(), "cache", "images", img)
+					fp := filepath.Join(localpath.MiniPath(), "cache", runtime.GOOS, "images", img)
```

Comment:
hmm, did we add some code changes for the images ? couldn't see any

Reply:
nope, i added that to the wrong spot. just pushed the fix.